src/main/java/com/mraof/minestuck/tileentity/redstone/RemoteComparatorTileEntity.java

```diff
@@ -0,0 +1,83 @@
+package com.mraof.minestuck.tileentity.redstone;
+
+import com.mraof.minestuck.MinestuckConfig;
+import com.mraof.minestuck.block.redstone.RemoteComparatorBlock;
+import com.mraof.minestuck.tileentity.MSTileEntityTypes;
+import net.minecraft.block.BlockState;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.network.NetworkManager;
+import net.minecraft.network.play.server.SUpdateTileEntityPacket;
+import net.minecraft.tileentity.ITickableTileEntity;
+import net.minecraft.tileentity.TileEntity;
+
+public class RemoteComparatorTileEntity extends TileEntity implements ITickableTileEntity
+{
+	private int tickCycle;
+	private static final int TICK_FREQUENCY = MinestuckConfig.SERVER.puzzleBlockTickRate.get();
```

Comment:
Since it's a config value, it might change during the course of the game. As such, you shouldn't really define a local constant like this. Use the config value directly instead.